src/components/InputDropdown/index.js

```diff
@@ -10,6 +10,7 @@ const DEFAULT = 'placeholder';
 function InputDropdown({
   customOnChange,
   inputProps,
+  isHalfWidth,
```

Comment:
An alternative to this prop would be passing a `passedStyles` or `className` prop. The downside to those (IMO) is that we would have to host styles in the `CreateFormFields` utility and pass those styles down. Ideally that utility wouldn't care about styles.